rating.html

```diff
@@ -1,17 +1,108 @@
 <!DOCTYPE html>
 <html>
-<body>
+<<head>
+    <!--Put  Tab Title Here-->
+    <title>Rate Movies</title>
+    <!--Set General Style Settings-->
+    <style>
 
-    <div class="displayMovie">
-        <h2> This is where the movie to choose will be inserted.</h2>
-        <body>
-        </body>
-    </div>
+        .content {
+            position: absolute;
+            top: 30%;
+            left: 50%;
+            -moz-transform: translateX(-50%) translateY(-50%);
+            -webkit-transform: translateX(-50%) translateY(-50%);
+            transform: translateX(-50%) translateY(-50%);
+            width: 80;
+            margin: auto;
+        }
 
-    <div class="ratingMechanism">
-        <h2> This section will contain the yes and no buttons to choose whether the user is interested in a movie or not.</h2>
-    </div>
```

Comment:
Same here, removing is fine as long as we have this heading elsewhere on the page in proper format, seen at line 56